<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="animation_lb">
        <property name="text">
         <string>动画标签</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="login_bottom" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item alignment="Qt::AlignLeft|Qt::AlignBottom">
          <widget class="QPushButton" name="register_btn">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;黑体&quot;;
</string>
           </property>
           <property name="text">
            <string>立即注册</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="pwd_le">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	border:none;
	border-bottom:2px solid darkgray;
	background-color:transparent;
	font: 25 12pt &quot;微软雅黑 Light&quot;;

}
QLineEdit:hover{
    border-bottom:2px solid gray;
}
QLineEdit:focus{
    border-bottom:2px solid rgb(90, 242, 255);
}
</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::PasswordEchoOnEdit</enum>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QComboBox" name="username_cmbx">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{
	border:none;
	border-bottom:2px solid darkgray;
	background-color:transparent;
	
	font: 25 12pt &quot;微软雅黑 Light&quot;;
}
QComboBox::drop-down {
	margin-top: 10px;
   	border-image: url(:/login/images/bottom_btn.png);
	background-color: transparent;
	width:30px;
	height:20px;
}
QComboBox:hover{
    border-bottom:2px solid gray;
}
QComboBox:focus{
    border-bottom:2px solid rgb(90, 242, 255);
}
</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>695989545</string>
            </property>
            <property name="icon">
             <iconset>
              <selectedon>:/login/images/safe1.png</selectedon>
             </iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7981146</string>
            </property>
            <property name="icon">
             <iconset>
              <selectedon>:/login/images/safe2.png</selectedon>
             </iconset>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="auto_login_cbx">
           <property name="text">
            <string>自动登录</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="login_btn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgb(220, 84, 71);
    border-radius: 15px;
	color:white;	
}
QPushButton:hover {
	background-color:rgb(254, 96, 81);
    	
}
QPushButton:pressed {
	background-color:rgb(157, 59, 50);
    	
}
QPushButton:disabled {
	background-color:gray;
	color:rgb(179, 179, 179);
    	
}</string>
           </property>
           <property name="text">
            <string>安全登录</string>
           </property>
           <property name="icon">
            <iconset>
             <selectedon>:/login/images/captain_america.png</selectedon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1" alignment="Qt::AlignRight">
          <widget class="QCheckBox" name="remember_cbx">
           <property name="text">
            <string>记住密码</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item alignment="Qt::AlignRight|Qt::AlignBottom">
          <widget class="QPushButton" name="url_btn">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/login/images/bd.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../login_register_calculate_case.qrc"/>
 </resources>
 <connections/>
</ui>
